create_gz_vendor_pkg/create_vendor_package.py

```diff
@@ -46,7 +46,7 @@
 
 # These dependencies will be removed from the package.xml provided by the upstream Gazebo library
 DEPENDENCY_DISALLOW_LIST = [
-    # python3-distutiol is not needed for CMake > 3.12. Also, it is currently failing to install on Noble
+    # python3-distutio is not needed for CMake > 3.12. Also, it is currently failing to install on Noble
```

Comment:
`python3-distutils`?

Reply:
Done in https://github.com/gazebo-tooling/gz_vendor/pull/8/commits/9286acf82a9c418a7a97f5d162b27410f1515444